Pachinko/Pachinko.c

```diff
@@ -0,0 +1,104 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+#define MINIMUM (100)
+#define LINESIZE (1024)
+
+/* Chooses a random number between 0 and 4 inclusively */
+void pickRandom(int* number)
+{
+	*number = rand() % 4;
+}
+
+/* Spins the slots */
+void playSlots(int slot[], size_t n)
+{
+	size_t i;
```

Comment:
Erase this and use n

Reply:
should i erase i, or use it to iterate, and then replace the > 3 with > n

Reply:
No thj sis a mistake. I was gonna say use n instead of 3.